configs/data/im2im/segmentation_plugin.yaml

```diff
@@ -33,14 +33,16 @@ transforms:
         reader:
           - _target_: cyto_dl.image.io.MonaiBioReader
             dimension_order_out: ${eval:'"CZYX" if ${spatial_dims}==3 else "CYX"'}
-            C: 0
+            # [TRAIN] Exposing C as a configurable param for multi-channel segs
+            C: ${target_col1_channel}
```

Comment:
`target_col1_channel` and `target_col2_channel` should also be added to the experiment config as top level keys with a reasonable default (either 0 or "MUST_OVERRIDE")

Reply:
I chose to use `0` as the default, since we used it previously.